core/modules-manager.php

```diff
@@ -123,6 +123,7 @@ public function get_modules_names() {
 			'content-sanitizer',
 			'editor-events',
 			'atomic-widgets',
+			'editor-global-classes',
```

Comment:
```suggestion
			'global-classes',
```
 
why did we add the `editor` prefix?

Reply:
Do you mean here or also in the class name?